zebra-state/src/service/check/anchors.rs

```diff
@@ -287,8 +300,8 @@ pub(crate) fn sprout_anchors_refer_to_treestates(
                     return Err(ValidateContextError::UnknownSproutAnchor {
                         anchor: joinsplit.anchor,
                         height,
-                        tx_index_in_block,
-                        transaction_hash: transaction_hashes[tx_index_in_block],
+                        tx_index_in_block: Some(tx_index_in_block),
```

Comment:
This should be `None` for mempool transactions.

Reply:
Fixed [here](https://github.com/ZcashFoundation/zebra/pull/5716/commits/55232d14135a0e82ae17d1c150223dbf4ded989d#diff-6c9234d1fb8c8dacc8e48e5ae36978d86fdbb86ff43e71498d0413af6b730738R280) in [`55232d1`](https://github.com/ZcashFoundation/zebra/pull/5716/commits/55232d14135a0e82ae17d1c150223dbf4ded989d).